src/MoBi.Presentation/Presenter/EditFavoritesPresenterInSimulation.cs

```diff
@@ -9,45 +9,51 @@
 
 namespace MoBi.Presentation.Presenter
 {
-   public interface IEditFavoritesInSimulationPresenter : IEditFavoritesPresenter<IMoBiSimulation>, IEditInSimulationPresenter
-   {
-      IEnumerable<IParameter> Favorites();
-   }
+    public interface IEditFavoritesInSimulationPresenter : IEditFavoritesPresenter<IMoBiSimulation>, IEditInSimulationPresenter
```

Comment:
There's probably an issue with the spacing in Visual Studio, or there was at the time when this change was made.

Can you do a Resharper cleanup and see if the spacing is fixed?